### PR TITLE
Add Donation links in navbar

### PR DESCRIPTION
**Pull request by Rohan-Rao**
Add 'Donation' links to the navbar. These links are hidden for small devices. Added the required language translations for these links
feature_DonationLink
feature_DonationLink_mobileView

.

### CODE REVIEW

**Comment by karngyan** [comment]
Hey @Rohan-Rao 

Thanks for the PR. Having a look.

File: lang/pa_IN.js [hunk 1]
```diff
@@ -1,30 +1,36 @@
...
     signIn: "ਸਾਈਨ ਇਨ ਕਰੋ",
-    signOut: "ਸਾਈਨ ਆਊਟ ਕਰੋ",
+    signOut: "ਸਾਈਨ ਆਊਟ",
+    donation: 'ਦਾਨ',
+    donationLinks: {
+      giveIndia: 'ਇੰਡੀਆ ਆਰਗੇਨਾਈਜ਼ੇਸ਼ਨ ਦਿਓ',
+      ketto: 'ਕੇਟੋ ਸੰਗਠਨ',
+      viratAnushkaFundraiser: 'ਵਿਰਾਟ ਅਤੇ ਅਨੁਸ਼ਕਾ ਫੰਡਰੇਜ਼ਰ'
+    },
   },
```
Comment (karngyan) [inline lang/pa_IN.js:15]: this file overwrote a lot of other strings. Can you have a look at this? @Rohan-Rao

**Review by karngyan — CHANGES_REQUESTED** [review]
Everything else looks great. Thanks man. Just check the Punjabi strings once. It overwrote a recent commit I believe.

File: lang/pa_IN.js (hunk 1 above)
Comment (Rohan-Rao) [reply lang/pa_IN.js:15]: yes, sure.

**Review by karngyan — APPROVED** [review]

**Pull request merged**